src/modules/hyprland/workspaces.cpp

```diff
@@ -1196,4 +1207,35 @@ void WindowCreationPayload::moveToWorksace(std::string &new_workspace_name) {
   m_workspaceName = new_workspace_name;
 }
 
+bool Workspaces::handleScroll(GdkEventScroll *e) {
+  if (gdk_event_get_pointer_emulated((GdkEvent *)e)) {
+    /**
+     * Ignore emulated scroll events on window
+     */
```

Comment:
```suggestion
    // Ignore emulated scroll events on window
```
I don't see why this should be a multiline comment